### DISCUSSION TITLE
Bring back proper markdown notes for new version of projects!

### DISCUSSION THREAD

**Discussion by krisu5** [community/community#34065] [Projects and Issues]
I'm disappointed seeing that "Projects" is more & more just "issues extended" type of tool ([here's one proof of that](https://github.com/community/community/discussions/30707)) and **not able to make proper markdown note cards**. Because markdown note cards are used by many people[^1][^2][^3][^4][^5][^6][^7][^8][^9][^10] <sup>(and more)</sup> and now because of removal of the function, new projects is actually more limited then classic projects.

And since the "new projects" doesn't have beta label anymore, I expect classic projects sunset in near future (it's not possible to currently make projects with old version, unless you already have one in repo / organization).

**And [you promised markdown notes for new projects, but it silently got removed from roadmap](https://github.com/community/community/discussions/4422#discussioncomment-2085342)** with no given reason.

I personally have **only used notes** in the classic projects, almost never listed issues with it and I found it reduntant for my workflow when ever I did so.

I tried the migration with [my repo's classic project](https://github.com/aegis-icons/aegis-icons/projects/1) and found out what a downgrade it was for my usage case.

All the cards become "drafts" (whatever those supposed to be, I don't know and don't care TBH) and now there's no way to interact or **even see the body of the cards** (yes, I know you can click to see the full info, but it still is massive downgrade compare how things were before).

At least **seeing the markdown body content and interacting it in the kanban board is must**, before I actually move from classic projects to new projects. **Doesn't matter if table mode doesn't have similar features**, I (and others) only care about kanban board

[^1]: https://github.com/community/community/discussions/4422
[^2]: https://github.com/community/community/discussions/6604
[^3]: https://github.com/community/community/discussions/20180
[^4]: https://github.com/community/community/discussions/5795
[^5]: https://github.com/community/community/discussions/7378
[^6]: https://github.com/community/community/discussions/6721
[^7]: https://github.com/community/community/discussions/7544
[^8]: https://github.com/community/community/discussions/14651
[^9]: https://github.com/community/community/discussions/7098
[^10]: https://github.com/community/community/discussions/17680

*(Sorry for the slightly annoyed tone of the comment, but I feel like left out as a user.)*

**Comment by krystof-k** [comment]
Yes, the old projects were great because it was possible to combine issues with notes or use just notes.
